### PR TITLE
GEOMESA-2351 Docs - upgrade guide for 2.1

### PR DESCRIPTION
**Pull request by elahrvivaz**
* Updated spark_version to 2.2.x or 2.3.x
* Added information on `--force` flag for ingest

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: docs/user/cli/ingest.rst [hunk 1]
```diff
@@ -40,6 +40,7 @@ Argument               Description
 ``-t, --threads``      Number of parallel threads used
 ``--input-format``     Format of input files (csv, tsv, avro, shp, json, etc)
 ``--run-mode``         Must be one of ``distributed`` or ``local``
+``--force``            Suppress any confirmation prompts
```
Comment (jnh5y) [inline docs/user/cli/ingest.rst]: Thanks for cleaning up after me:).

File: docs/user/upgrade.rst [hunk 2]
```diff
@@ -40,18 +39,26 @@ fully supports data written with version 1.2.2 or later, and mostly supports dat
...
-Distributed Runtime Compatibility
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+API Compatibility
+^^^^^^^^^^^^^^^^^
 
-For data stores with a distributed component (e.g. Accumulo or HBase), distributed runtime compatibility refers
-to the ability to write or query data with a client that is a different version than the distributed code.
-Similarly, this also covers the ability to have different versions of the distributed code on different machines in
-a single cluster.
+The GeoMesa public API is not currently well defined, so API compatibility is only guaranteed at the DataStore
```
Comment (jnh5y) [inline docs/user/upgrade.rst]: Should we say "GeoTools DataStore" interface and link to some of the GeoTools docs?

**Comment by jnh5y** [comment]
Not a requirement, but do we have a good place in the documentation to 'brag' / advertise the new features?  I suppose I'm looking for a place to capture the 'Notable Features' from https://geomesa.atlassian.net/wiki/spaces/GEOMESA/pages/465174529/GeoMesa+2.1.0+Release+Notes+preview on the website/documentation.

File: docs/user/upgrade.rst (hunk 2 above)
Comment (elahrvivaz) [reply docs/user/upgrade.rst]: wilco

**Comment by elahrvivaz** [comment]
We have a link to the release notes on the main webpage under the documentation menu, but nothing in the actual docs themselves - do you think that's good enough?

File: docs/user/cli/ingest.rst (hunk 1 above)
Comment (elahrvivaz) [reply docs/user/cli/ingest.rst]: :+1:

**Comment by jnh5y** [comment]
Hmm...  I wonder if people will hit the links or if they'd appreciate/benefit from a copy in the docs.  Also, if we had high-level notable features from multiple releases on the same page, it might be useful/motivational.

**Comment by elahrvivaz** [comment]
added a ticket to add it: https://geomesa.atlassian.net/browse/GEOMESA-2359

**Pull request merged**